### PR TITLE
feat: making kp async

### PR DESCRIPTION
**Pull request by sumit-158**
### What
- Making the kp async 

### Fixes bug(s)
- #22

### CODE REVIEW

File: tests/test_knowledge_panels.py [hunk 1]
```diff
@@ -415,7 +420,7 @@ def test_data_quality_kp_with_all_tags(monkeypatch):
     }
 
 
-def test_last_edits_kp_with_one_facet_and_value(monkeypatch):
+async def test_last_edits_kp_with_one_facet_and_value(monkeypatch):
```
Comment (alexgarel) [inline tests/test_knowledge_panels.py:423]: Did you install an [async plugin](https://docs.pytest.org/en/latest/reference/plugin_list.html?highlight=async) for pytest ?

If yes you should add it to requirements.

Comment (sumit-158) [reply tests/test_knowledge_panels.py:423]: yes, pytest-asyncio, sorry I forget to add!

File: tests/test_main.py [hunk 2]
```diff
@@ -14,17 +14,17 @@ def test_hello():
...
+async def test_knowledge_panel_badendpoint():
     response = client.get("/knowledge_panel_bad")
     assert response.status_code == 404
 
 
-def test_knowledge_panel_with_facet():
+async def test_knowledge_panel_with_facet():
     response = client.get(
         "/knowledge_panel?facet_tag=packaging&value_tag=plastic-box&country=Germany"
     )
```
[the remark this answers is not in the document]
Comment (sumit-158) [reply tests/test_main.py:30]: yes! otherwise tests fail. 
https://pypi.org/project/pytest-asyncio/

[the remark this answers is not in the document]
Comment (sumit-158) [reply tests/test_main.py:30]: Sorry, you are right! It didn't need async :-)

**Review by alexgarel — APPROVED** [review]
Let's merge :-)

**Pull request merged**